lib/components/narrative/trip-tools.js

```diff
@@ -169,7 +171,20 @@ LinkButton.propTypes = {
   url: PropTypes.string
 }
 
+const makeIcon = memoize((SvgIcon, iconType) => {
```

Comment:
By default, lodash's memoize only caches based on the first argument provided to the function. Might that be a problem here? 
> By default, the first argument provided to the memoized function is used as the map cache key.

https://lodash.com/docs/#memoize

Reply:
Good catch! Also tricky. If I pass an object such as `{ icon: SvgIcon, type: popupTarget }`, the value that determines the hash changes at every render and makes memoizing moot. Interestingly, if only SvgIcon gets memoized, the console warning regarding creating a dynamic component still appears only once, which is the desired outcome.